pyvista/utilities/features.py

```diff
@@ -116,3 +117,55 @@ def transform_vectors_sph_to_cart(theta, phi, r, u, v, w):
     w_t = np.cos(ph) * w - np.sin(ph) * v
 
     return u_t, v_t, w_t
+
+
+def extrude(mesh, vector):
+    """Sweep polygonal data creating a "skirt" from free edges and
+    lines, and lines from vertices.
+
+    This takes polygonal data as input and generates polygonal
+    data on output. The input dataset is swept according to some
+    extrusion function and creates new polygonal primitives. These
+    primitives form a "skirt" or swept surface. For example,
+    sweeping a line results in a quadrilateral, and sweeping a
+    triangle creates a "wedge".
+
+    There are a number of control parameters for this filter. You
+    can control whether the sweep of a 2D object (i.e., polygon or
+    triangle strip) is capped with the generating geometry via the
+    "Capping" parameter.
+
+    The skirt is generated by locating certain topological
+    features. Free edges (edges of polygons or triangle strips
+    only used by one polygon or triangle strips) generate
+    surfaces. This is true also of lines or polylines. Vertices
+    generate lines.
+
+    This filter can be used to create 3D fonts, 3D irregular bar
+    charts, or to model 2 1/2D objects like punched plates. It
+    also can be used to create solid objects from 2D polygonal
+    meshes.
+
+    Parameters
+    ----------
+    mesh : pyvista.PolyData
+        Mesh to extrude.
+
+    vector : np.ndarray or list
+        Direction and length to extrude the mesh in.
+
+    Examples
+    --------
+    Extrude a half arc circle
+
+    >>> import pyvista
+    >>> arc = pyvista.CircularArc([-1, 0, 0], [1, 0, 0], [0, 0, 0])
+    >>> mesh = pyvista.extrude(arc, [0, 0, 1])
+    >>> mesh.plot()  # doctest:+SKIP
+    """
+    extrude_filter = vtk.vtkLinearExtrusionFilter()
+    extrude_filter.SetExtrusionTypeToVectorExtrusion()
+    extrude_filter.SetVector(*vector)
+    extrude_filter.SetInputData(mesh)
+    extrude_filter.Update()
+    return pyvista.wrap(extrude_filter.GetOutput())
```

Comment:
Considering that `vtkLinearExtrusionFilter` is a `vtkPolyDataAlgorithm` subclass, I think this filtering routine should be implemented under the `pyvista.core.filters.PolyDataFilters` class and only defined once. This would avoid having the docs written twice and would still make the algorithm accesible from any `PolyData` mesh

Reply:
fixed in e37c761748d03b0651c468bad6ec687cce8d2276